plugin.py

```diff
@@ -3,7 +3,7 @@
 # Author: thomas-villagers
 #
 """
-<plugin key="YamahaPlug" name="Yamaha AV Receiver" author="thomasvillagers" version="2.0.0" wikilink="http://www.domoticz.com/wiki/plugins/plugin.html" externallink="https://yamaha.com/products/audio_visual/av_receivers_amps/">
+<plugin key="YamahaPlug" name="Yamaha AV Receiver with Kodi Remote" author="thomasvillagers" version="2.0.0" wikilink="http://www.domoticz.com/wiki/plugins/plugin.html" externallink="https://yamaha.com/products/audio_visual/av_receivers_amps/">
```

Comment:
It is hardcoded in domoticz.js. The remote control popup is only showed when the device has a name contains:

- Kodi
- Panasonic
- Logitech Media Server
- HEOS by DENON